[Product Partnerships] [Project Sleek] Updated Project Configurations For EKS Add-ons

rahul188 opened the pull request:
We've taken steps to update files related to secrets & NewRelic-licence-key
- Specifically, we have securely removed the below files -
  -  aws_mp_configuration_schema.json
  - secrets.yaml
  - values.schema.json.
- Additionally, we've updated the version information in chart.yml to ensure accurate tracking of our application's state.
- In an effort to streamline our configurations, we've also eliminated unnecessary parameters from values.yml.

rahul188 committed:
[Product Partnerships] [Project Sleek] Updated Project Configurations

xqi-nr commented on charts/newrelic-k8s-operator/values.yaml:
```diff
@@ -1,12 +1,11 @@
 version: ""
-newRelicLicenseKey: ""
```
I notice you remove the license key, then how will the integration get the key from?

rahul188 replied:
We have license key inside [minimal_nribundle](https://github.com/newrelic/newrelic-k8s-operator/blob/main/config/samples/minimal_nribundle.yaml) as well. 
so the customer will First create NR-Operator and later Apply [CRD](https://github.com/newrelic/newrelic-k8s-operator/blob/7a64f4e178eddb42125272d2d800301698c25f74/config/samples/minimal_nribundle.yaml#L8).

xqi-nr replied:
[minimal_nribundle](https://github.com/newrelic/newrelic-k8s-operator/blob/main/config/samples/minimal_nribundle.yaml) is just a sample yaml file. It cannot replace the chart's default values.yaml right?

rahul188 replied:
putting license key in minimal_nribundle will help creating configuration less project. putting values in yaml file and then applying minimal_nribundle will not be of any use as minimal_nri bundle will override all the values which is a direction towards configuration less project.

xqi-nr replied:
but for the operator customers, the chart values.yaml will be default place for them to do configuraion right?

xqi-nr commented on charts/newrelic-k8s-operator/aws_mp_configuration_schema.json:
```diff
@@ -1,11 +0,0 @@
-{
```
Is this schema and the secret below remove for security purpose? Why?

rahul188 replied:
Yes, it's removed for security purposes which is raised by Amazon.

rahul188 replied:
As we have made this project configuration-less so [aws_mp_configuration_schema.json](https://github.com/newrelic/newrelic-k8s-operator/pull/63/files/b3d936f55b066f2be6f4a5280ec211cf9ce07723#diff-a4e1a495a329b0555d0868737ba88a46fcb4e8978668be9fa7a463c30826c4e2) is no longer needed.

xqi-nr commented on charts/newrelic-k8s-operator/values.yaml:
```diff
@@ -1,12 +1,11 @@
 nri-prometheus:
   # nri-prometheus.enabled -- Install the [`nri-prometheus` chart](https://github.com/newrelic/nri-prometheus/tree/main/charts/nri-prometheus)
-  enabled: false
+  enabled: true
```
you do not need to change the default values, but set whatever values you need

rahul188 replied:
We have tried to change those values in CRDs but that is not reflecting while doing the deployments. so we have marked them as **True** in values.yml as a temporary fix. later will deep dive & try to improvise it.

xqi-nr replied:
please open a jira to track the further deep dive

rahul188 replied:
@xqi-nr, Could you please share the Jira URL and grant me access?

xqi-nr replied:
@rahul188 you can login to okta and click **Jira software**

Do you team use Jira or not? If not, I can create the jira for you

juanjjaramillo replied:
@xqi-nr I agree. Regardless of what installation method the user chooses, the experience should be the same. @rahul188 please use same default values as in `nri-bundle`

aj-NR replied:
done

xqi-nr commented on charts/newrelic-k8s-operator/values.yaml:
```diff
@@ -15,53 +14,41 @@ nri-metadata-injection:
 global:
-  # -- The cluster name for the Kubernetes cluster.
-  cluster: ""
-
```
The same comments as above. Do not modify the default values.yaml unless necessary.

juanjjaramillo replied:
Where are we supplying the values that are removed here? In `minimal_nribundle.yaml`? I do not see all values here:
```
apiVersion: newrelic.com/v1alpha1
kind: NRIBundle
metadata:
  name: nribundle-minimal
spec:
  global:
    cluster: "k8scluster126"
    licenseKey: "<my key>"
    lowDataMode: true
  nri-kube-events:
    enabled: true
  newrelic-logging:
    enabled: true
    lowDataMode: true
  newrelic-infrastructure:
    enabled: true
    privileged: true
  newrelic-pixie:
    apiKey: "<my key>"
    enabled: false
  newrelic-prometheus-agent:
    config:
      kubernetes:
        integrations_filter:
          enabled: false
    enabled: true
    lowDataMode: true
  pixie-chart:
    clusterName: "k8scluster126"
    deployKey: "<my key>"
    enabled: false
```

xqi-nr replied:
The above is just an example for Rahul

The complete command to install the operator and nribundle are here https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/installation/install-with-operator/ through operator chart

@rahul188 somehow, I cannot repro my success on creating the NRIBundle CR  :(

juanjjaramillo commented on charts/newrelic-k8s-operator/templates/secrets.yaml:
```diff
@@ -1,13 +0,0 @@
-apiVersion: v1
```
@rahul188 : 
1. Can you please document why you are deleting this file?
2. What is the alternative that replaces the use of this file?

juanjjaramillo committed:
ci: Introduce CHANGELOG.md (#66)

Introduce CHANGELOG.md to keep track of repo changes

juanjjaramillo committed:
ci: Add pull request template (#67)

juanjjaramillo committed:
ci: Add 'changelog' workflow (#68)

## Description
<!-- Please include a summary of the change in your PR and what it's
fixing. -->
Enforce changelog changes in PRs

## Type of change
<!-- Please check the relevant option. -->

- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)
- [x] New feature / enhancement (non-breaking change which adds
functionality)
- [ ] Security fix
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist:
<!-- Please check applicable options. -->

- [x] Add changelog entry following the [contributing
guide](../CONTRIBUTING.md#pull-requests)
- [ ] Documentation has been updated
- [ ] This change requires changes in testing:
  - [ ] unit tests
  - [ ] E2E tests

juanjjaramillo committed:
ci: Enable automatic release (#69)

## Description
<!-- Please include a summary of the change in your PR and what it's
fixing. -->
Enable automatic release

## Type of change
<!-- Please check the relevant option. -->

- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)
- [x] New feature / enhancement (non-breaking change which adds
functionality)
- [ ] Security fix
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist:
<!-- Please check applicable options. -->

- [x] Add changelog entry following the [contributing
guide](../CONTRIBUTING.md#pull-requests)
- [ ] Documentation has been updated
- [ ] This change requires changes in testing:
  - [ ] unit tests
  - [ ] E2E tests

renovate[bot] committed:
chore(deps): update alpine docker tag to v3.18.4 (#51)

[![Mend
Renovate](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)

This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| alpine | final | patch | `3.18.2` -> `3.18.4` |

---

### Configuration

📅 **Schedule**: Branch creation - At any time (no schedule defined),
Automerge - At any time (no schedule defined).

🚦 **Automerge**: Enabled.

♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the
rebase/retry checkbox.

🔕 **Ignore**: Close this PR and you won't be reminded about this update
again.

---

- [ ] <!-- rebase-check -->If you want to rebase/retry this PR, check
this box

---

This PR has been generated by [Mend
Renovate](https://www.mend.io/free-developer-tools/renovate/). View
repository job log
[here](https://developer.mend.io/github/newrelic/newrelic-k8s-operator).

<!--renovate-debug:eyJjcmVhdGVkSW5WZXIiOiIzNi4yNy4xIiwidXBkYXRlZEluVmVyIjoiMzcuMC4zIiwidGFyZ2V0QnJhbmNoIjoibWFpbiJ9-->

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update actions/checkout action to v4 (#58)

[![Mend
Renovate](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)

This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [actions/checkout](https://togithub.com/actions/checkout) | action |
major | `v3` -> `v4` |

---

### Release Notes

<details>
<summary>actions/checkout (actions/checkout)</summary>

###
[`v4`](https://togithub.com/actions/checkout/blob/HEAD/CHANGELOG.md#v400)

[Compare Source](https://togithub.com/actions/checkout/compare/v3...v4)

- [Support fetching without the --progress
option](https://togithub.com/actions/checkout/pull/1067)
-   [Update to node20](https://togithub.com/actions/checkout/pull/1436)

</details>

---

### Configuration

📅 **Schedule**: Branch creation - At any time (no schedule defined),
Automerge - At any time (no schedule defined).

🚦 **Automerge**: Disabled by config. Please merge this manually once you
are satisfied.

♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the
rebase/retry checkbox.

🔕 **Ignore**: Close this PR and you won't be reminded about this update
again.

---

- [ ] <!-- rebase-check -->If you want to rebase/retry this PR, check
this box

---

This PR has been generated by [Mend
Renovate](https://www.mend.io/free-developer-tools/renovate/). View
repository job log
[here](https://developer.mend.io/github/newrelic/newrelic-k8s-operator).

<!--renovate-debug:eyJjcmVhdGVkSW5WZXIiOiIzNi43OC44IiwidXBkYXRlZEluVmVyIjoiMzYuNzguOCIsInRhcmdldEJyYW5jaCI6Im1haW4ifQ==-->

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Commit:
Correct broken docs link (#54)

Co-authored-by: Juan Jose Jaramillo <juanjjaramillo@yahoo.com>

renovate[bot] committed:
chore(deps): update golang version (#70)

[![Mend
Renovate](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)

This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [go](https://go.dev/) ([source](https://togithub.com/golang/go)) |
golang | minor | `1.20` -> `1.21` |
| golang | stage | minor | `1.20-alpine` -> `1.21-alpine` |

---

### Release Notes

<details>
<summary>golang/go (go)</summary>

###
[`v1.21.2`](https://togithub.com/golang/go/compare/go1.21.1...go1.21.2)

###
[`v1.21.1`](https://togithub.com/golang/go/compare/go1.21.0...go1.21.1)

###
[`v1.21.0`](https://togithub.com/golang/go/compare/go1.20.8...go1.21.0)

###
[`v1.20.9`](https://togithub.com/golang/go/compare/go1.20.8...go1.20.9)

###
[`v1.20.8`](https://togithub.com/golang/go/compare/go1.20.7...go1.20.8)

###
[`v1.20.7`](https://togithub.com/golang/go/compare/go1.20.6...go1.20.7)

###
[`v1.20.6`](https://togithub.com/golang/go/compare/go1.20.5...go1.20.6)

###
[`v1.20.5`](https://togithub.com/golang/go/compare/go1.20.4...go1.20.5)

###
[`v1.20.4`](https://togithub.com/golang/go/compare/go1.20.3...go1.20.4)

###
[`v1.20.3`](https://togithub.com/golang/go/compare/go1.20.2...go1.20.3)

###
[`v1.20.2`](https://togithub.com/golang/go/compare/go1.20.1...go1.20.2)

###
[`v1.20.1`](https://togithub.com/golang/go/compare/go1.20...go1.20.1)

</details>

---

### Configuration

📅 **Schedule**: Branch creation - At any time (no schedule defined),
Automerge - At any time (no schedule defined).

🚦 **Automerge**: Disabled by config. Please merge this manually once you
are satisfied.

♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the
rebase/retry checkbox.

👻 **Immortal**: This PR will be recreated if closed unmerged. Get
[config help](https://togithub.com/renovatebot/renovate/discussions) if
that's undesired.

---

- [ ] <!-- rebase-check -->If you want to rebase/retry this PR, check
this box

---

This PR has been generated by [Mend
Renovate](https://www.mend.io/free-developer-tools/renovate/). View
repository job log
[here](https://developer.mend.io/github/newrelic/newrelic-k8s-operator).

<!--renovate-debug:eyJjcmVhdGVkSW5WZXIiOiIzNy4wLjMiLCJ1cGRhdGVkSW5WZXIiOiIzNy4wLjMiLCJ0YXJnZXRCcmFuY2giOiJtYWluIn0=-->

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Juan Jose Jaramillo <juanjjaramillo@yahoo.com>

renovate[bot] committed:
fix(deps): update module github.com/onsi/gomega to v1.28.0 (#50)

[![Mend
Renovate](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)

This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [github.com/onsi/gomega](https://togithub.com/onsi/gomega) | require |
minor | `v1.27.8` -> `v1.28.0` |

---

### Release Notes

<details>
<summary>onsi/gomega (github.com/onsi/gomega)</summary>

### [`v1.28.0`](https://togithub.com/onsi/gomega/releases/tag/v1.28.0)

[Compare
Source](https://togithub.com/onsi/gomega/compare/v1.27.10...v1.28.0)

#### 1.28.0

##### Features

- Add VerifyHost handler to ghttp
([#&#8203;698](https://togithub.com/onsi/gomega/issues/698))
\[[`0b03b36`](https://togithub.com/onsi/gomega/commit/0b03b36)]

##### Fixes

- Read Body for Newer Responses in HaveHTTPBodyMatcher
([#&#8203;686](https://togithub.com/onsi/gomega/issues/686))
\[[`18d6673`](https://togithub.com/onsi/gomega/commit/18d6673)]

##### Maintenance

- Bump github.com/onsi/ginkgo/v2 from 2.11.0 to 2.12.0
([#&#8203;693](https://togithub.com/onsi/gomega/issues/693))
\[[`55a33f3`](https://togithub.com/onsi/gomega/commit/55a33f3)]
- Typo in matchers.go
([#&#8203;691](https://togithub.com/onsi/gomega/issues/691))
\[[`de68e8f`](https://togithub.com/onsi/gomega/commit/de68e8f)]
- Bump commonmarker from 0.23.9 to 0.23.10 in /docs
([#&#8203;690](https://togithub.com/onsi/gomega/issues/690))
\[[`ab17f5e`](https://togithub.com/onsi/gomega/commit/ab17f5e)]
- chore: update test matrix for Go 1.21
([#&#8203;689](https://togithub.com/onsi/gomega/issues/689))
\[[`5069017`](https://togithub.com/onsi/gomega/commit/5069017)]
- Bump golang.org/x/net from 0.12.0 to 0.14.0
([#&#8203;688](https://togithub.com/onsi/gomega/issues/688))
\[[`babe25f`](https://togithub.com/onsi/gomega/commit/babe25f)]

### [`v1.27.10`](https://togithub.com/onsi/gomega/releases/tag/v1.27.10)

[Compare
Source](https://togithub.com/onsi/gomega/compare/v1.27.9...v1.27.10)

#### 1.27.10

##### Fixes

- fix: go 1.21 adding goroutine ID to creator+location
([#&#8203;685](https://togithub.com/onsi/gomega/issues/685))
\[[`bdc7803`](https://togithub.com/onsi/gomega/commit/bdc7803)]

###
[`v1.27.9`](https://togithub.com/onsi/gomega/compare/v1.27.8...v1.27.9)

[Compare
Source](https://togithub.com/onsi/gomega/compare/v1.27.8...v1.27.9)

</details>

---

### Configuration

📅 **Schedule**: Branch creation - At any time (no schedule defined),
Automerge - At any time (no schedule defined).

🚦 **Automerge**: Enabled.

♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the
rebase/retry checkbox.

🔕 **Ignore**: Close this PR and you won't be reminded about this update
again.

---

- [ ] <!-- rebase-check -->If you want to rebase/retry this PR, check
this box

---

This PR has been generated by [Mend
Renovate](https://www.mend.io/free-developer-tools/renovate/). View
repository job log
[here](https://developer.mend.io/github/newrelic/newrelic-k8s-operator).

<!--renovate-debug:eyJjcmVhdGVkSW5WZXIiOiIzNi4xMS4wIiwidXBkYXRlZEluVmVyIjoiMzcuMC4zIiwidGFyZ2V0QnJhbmNoIjoibWFpbiJ9-->

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Juan Jose Jaramillo <juanjjaramillo@yahoo.com>

renovate[bot] committed:
fix(deps): update module github.com/onsi/ginkgo/v2 to v2.12.1 (#41)

[![Mend
Renovate](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)

This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [github.com/onsi/ginkgo/v2](https://togithub.com/onsi/ginkgo) |
require | patch | `v2.12.0` -> `v2.12.1` |

---

### Release Notes

<details>
<summary>onsi/ginkgo (github.com/onsi/ginkgo/v2)</summary>

### [`v2.12.1`](https://togithub.com/onsi/ginkgo/releases/tag/v2.12.1)

[Compare
Source](https://togithub.com/onsi/ginkgo/compare/v2.12.0...v2.12.1)

#### 2.12.1

##### Fixes

- Print logr prefix if it exists
([#&#8203;1275](https://togithub.com/onsi/ginkgo/issues/1275))
\[[`90d4846`](https://togithub.com/onsi/ginkgo/commit/90d4846)]

##### Maintenance

- Bump actions/checkout from 3 to 4
([#&#8203;1271](https://togithub.com/onsi/ginkgo/issues/1271))
\[[`555f543`](https://togithub.com/onsi/ginkgo/commit/555f543)]
- Bump golang.org/x/sys from 0.11.0 to 0.12.0
([#&#8203;1270](https://togithub.com/onsi/ginkgo/issues/1270))
\[[`d867b7d`](https://togithub.com/onsi/ginkgo/commit/d867b7d)]

</details>

---

### Configuration

📅 **Schedule**: Branch creation - At any time (no schedule defined),
Automerge - At any time (no schedule defined).

🚦 **Automerge**: Enabled.

♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the
rebase/retry checkbox.

🔕 **Ignore**: Close this PR and you won't be reminded about this update
again.

---

- [ ] <!-- rebase-check -->If you want to rebase/retry this PR, check
this box

---

This PR has been generated by [Mend
Renovate](https://www.mend.io/free-developer-tools/renovate/). View
repository job log
[here](https://developer.mend.io/github/newrelic/newrelic-k8s-operator).

<!--renovate-debug:eyJjcmVhdGVkSW5WZXIiOiIzNS4xMDIuMTAiLCJ1cGRhdGVkSW5WZXIiOiIzNy4wLjMiLCJ0YXJnZXRCcmFuY2giOiJtYWluIn0=-->

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

juanjjaramillo committed:
fix: Fix repolinter error (#71)

## Description
<!-- Please include a summary of the change in your PR and what it's
fixing. -->

## Type of change
<!-- Please check the relevant option. -->

- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)
- [ ] New feature / enhancement (non-breaking change which adds
functionality)
- [ ] Security fix
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:
<!-- Please check applicable options. -->

- [ ] Add changelog entry following the [contributing
guide](../CONTRIBUTING.md#pull-requests)
- [ ] Documentation has been updated
- [ ] This change requires changes in testing:
  - [ ] unit tests
  - [ ] E2E tests

AmitBenAmi committed:
fix(rbac): adding the `patch` operation for `operators.coreos.com` (#56)

The `patch` operation was missing, causing numerous errors in the
operator in the form of:
```
newrelic-k8s-operator-controller-manager-6b4487bb-mk2cc manager 2023-08-23T15:35:31Z	ERROR	Reconciler error	{"controller": "nribundle-controller", "object": {"name":"nribundle","namespace":"newrelic"}, "namespace": "newrelic", "name": "nribundle", "reconcileID": "0343110a-3310-4d93-a7d8-07b4009df98e", "error": "patch error: catalogsources.operators.coreos.com \"pixie-operator-index\" is forbidden: User \"system:serviceaccount:newrelic:newrelic-k8s-operator-controller-manager\" cannot patch resource \"catalogsources\" in API group \"operators.coreos.com\" in the namespace \"px-operator\""}
newrelic-k8s-operator-controller-manager-6b4487bb-mk2cc manager sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler
newrelic-k8s-operator-controller-manager-6b4487bb-mk2cc manager 	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.1/pkg/internal/controller/controller.go:329
newrelic-k8s-operator-controller-manager-6b4487bb-mk2cc manager sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
newrelic-k8s-operator-controller-manager-6b4487bb-mk2cc manager 	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.1/pkg/internal/controller/controller.go:274
newrelic-k8s-operator-controller-manager-6b4487bb-mk2cc manager sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
newrelic-k8s-operator-controller-manager-6b4487bb-mk2cc manager 	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.1/pkg/internal/controller/controller.go:235
```

fixes newrelic/newrelic-k8s-operator#55

---------

Signed-off-by: Amit Ben Ami <benami.amit1@gmail.com>
Co-authored-by: Juan Jose Jaramillo <juanjjaramillo@yahoo.com>

juanjjaramillo commented on charts/newrelic-k8s-operator/Chart.yaml:
```diff
@@ -21,7 +21,7 @@ type: application
 # version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.4
+version: 0.1.6
```
@rahul188 please remove this entry. We now have in place automated releases for this repo. You will also need to rebase your branch since we now enforce that branches have to be up to date before merging

rahul188 replied:
@juanjjaramillo, we have onboarded to EKS with this version that is why there is necessity to update the version here. i suppose there is no pipeline between AWS & current repo which can update the version automatically.

NewRelic-K8s-Agents-Bot committed:
Update changelog with changes from v0.1.0

juanjjaramillo committed:
feat: Sync versions between app image and Helm chart to ease customer inception (#73)

## Description
<!-- Please include a summary of the change in your PR and what it's
fixing. -->
It is hard to track the mapping between `version` and `appVersion` on a
Helm chart. By triggering a new image release, we are forcing that the
next chart release will sync both versions to be the same.

## Type of change
<!-- Please check the relevant option. -->

- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)
- [x] New feature / enhancement (non-breaking change which adds
functionality)
- [ ] Security fix
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist:
<!-- Please check applicable options. -->

- [x] Add changelog entry following the [contributing
guide](../CONTRIBUTING.md#pull-requests)
- [x] Documentation has been updated
- [ ] This change requires changes in testing:
  - [ ] unit tests
  - [ ] E2E tests

NewRelic-K8s-Agents-Bot committed:
Update changelog with changes from v0.2.0

github-actions[bot] committed:
Bump version and update docs (#74)

Bump version and appVersion and results of running helm docs as part of
release automation.

Co-authored-by: juanjjaramillo <juanjjaramillo@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update module github.com/onsi/ginkgo/v2 to v2.13.0

NewRelic-K8s-Agents-Bot committed:
Update changelog with changes from v0.2.1

svetlanabrennan committed:
Update release.yml

aj-NR committed:
Reverting values.yaml as discussed with Jerry & Mikhail

aj-NR committed:
Merge branch 'newrelic:main' into aws-k8s-operator-testing

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/newrelic/newrelic-k8s-operator?pullRequest=63) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/newrelic/newrelic-k8s-operator?pullRequest=63) before we can accept your contribution.<br/>**6** out of **7** committers have signed the CLA.<br/><br/>:white_check_mark: rahul188<br/>:white_check_mark: aj-NR<br/>:white_check_mark: AmitBenAmi<br/>:white_check_mark: svetlanabrennan<br/>:white_check_mark: NewRelic-K8s-Agents-Bot<br/>:white_check_mark: juanjjaramillo<br/>:x: renovate[bot]<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/newrelic/newrelic-k8s-operator?pullRequest=63) it.</sub>

aj-NR committed:
Update Chart.yaml

aj-NR committed:
Update Chart.yaml

Updating version back to original

aj-NR committed:
Merge branch 'main' into aws-k8s-operator-testing
